Spanner/src/Operation.php

```diff
@@ -207,6 +214,7 @@ public function rollback(Session $session, $transactionId, array $options = [])
      *         [the upstream documentation](https://cloud.google.com/spanner/docs/reference/rest/v1/RequestOptions).
      *         Please note, if using the `priority` setting you may utilize the constants available
      *         on {@see \Google\Cloud\Spanner\V1\RequestOptions\Priority} to set a value.
+     *     @type Transaction transactionHandle Transaction to be used for this operation.
```

Comment:
nit: just call this `$transaction` instead of `$transactionHandle`

Reply:
This hasn't been updated though

Reply:
Resolving this because it's covered in https://github.com/googleapis/google-cloud-php/pull/6810